docs/interaction/rpc.md

```diff
@@ -6,122 +6,675 @@ sidebar_label: RPC
 
 Standard JSON RPC 2.0 is used across the board.
 
-The following methods are available:
+Notes:
+
+- all methods are HTTP `POST`
+- endpoint URL varies by network
+  - for TestNet use `https://rpc.nearprotocol.com`
+  - for Staging use `https://staging-rpc.nearprotocol.com`
```

Comment:
Let's mention that this environment is not supposed to be stable. It may break at any moment. (do we even want an unstable environment in public docs?)

Reply:
i added a short message about instability.  the only reason i included it here is bc sometimes TestNet is down and the only way to get an RPC response (for testing, learning, etc) is via staging.